src/NServiceBus.Transport.RabbitMQ.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -73,4 +73,8 @@ namespace NServiceBus.Transport.RabbitMQ
         System.Threading.Tasks.ValueTask SetupSubscription(RabbitMQ.Client.IChannel channel, NServiceBus.Unicast.Messages.MessageMetadata type, string subscriberName, System.Threading.CancellationToken cancellationToken = default);
         System.Threading.Tasks.ValueTask TeardownSubscription(RabbitMQ.Client.IChannel channel, NServiceBus.Unicast.Messages.MessageMetadata type, string subscriberName, System.Threading.CancellationToken cancellationToken = default);
     }
+    public static class NativeIntegrationDispatchPropertiesExtensions
```

Comment:
Quick question:

This PR describes a native integration scenario, but as far as other native rabbitmq libraries are concerned it doesn't really care about the content type property at all. Is this "native integration" specific to a specific integration scenario?

It also seems that there are some other "native integration" related scenarios popping up that require some low level dispatcher / receiver configuration but the domain specific language is not aligned for the extension methods etc.

https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/pull/1072